go/callbacks.go

```diff
@@ -10,6 +10,9 @@ package main
 
 /*
 struct GraphQLAstField;
+
+extern int printField(struct GraphQLAstField *field, void *unused);
```

Comment:
`extern` on a function prototype does nothing. Can you remove it, please?

Reply:
emm...you are right.